cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -3943,6 +3943,19 @@ TEST(TestArrowReaderAdHoc, WriteBatchedNestedNullableStringColumn) {
   ::arrow::AssertTablesEqual(*expected, *actual, /*same_chunk_layout=*/false);
 }
 
+TEST(TestArrowReaderAdHoc, OldDataPageV2) {
+  // ARROW-17100
+  const char* c_root = std::getenv("ARROW_TEST_DATA");
+  if (!c_root) {
+    GTEST_SKIP() << "ARROW_TEST_DATA not set.";
+  }
```

Comment:
I think the feeling was it was Arrow C++ specific because it's not *technically* a valid parquet file but we wanted to be able to read it anyways for backwards compatibility. My understanding of the `parquet-testing` repo was it contains files that all readers should either definitely support or definitely not. But I'm open to moving if I'm misunderstanding the purpose of `parquet-testing`.

Reply:
Ah, ok - sounds good then